tests/modules/geometry-calculation.test.js

```diff
@@ -0,0 +1,111 @@
+const { calculateDistance, calculateJunctionPoint, isPointOnLine } = require('../../modules/geometry-calculation')
+
+
+const Point = jest.requireActual('../../modules/ecs6-class/point');
+const Line = jest.requireActual('../../modules/ecs6-class/line')
+
+/////האם זה בכיוון???????
+jest.mock('../../modules/ecs6-class/line', () => {
+    return jest.fn().mockImplementation(({ point1, point2, n, slope }) => {
```

Comment:
This is a good way 
now check that the function gets here and not in the real functions, log data to the console

Reply:
It really doesn't go to me for the functions in the mock, but how it will go there? - I don't know.

Reply:
In the way I did the mock for the constructors, where do the tests go? To the real constructors?

Reply:
did you try to write a log to the console in the constructors to see which constructor was run?
use the answer from https://stackoverflow.com/questions/40465047/how-can-i-mock-an-es6-module-import-using-jest to figure out the mocks